pass/Easy.cpp

```diff
@@ -128,6 +128,17 @@ namespace easy {
           }
         }
       }
+      if(auto *GEP = dyn_cast<GetElementPtrInst>(V)) {
+        return mayAlias(GEP->getPointerOperand(), GO);
+      }
+      if(auto* OtherGV = dyn_cast<GlobalVariable>(V)) {
+        if(!OtherGV->hasInitializer())
+          return false;
+        return mayAlias(OtherGV->getInitializer(), GO);
```

Comment:
Similar to the Alloca case, shouldn't we also look the StoreInst that take OtherGV as the pointer parameter?